rdkit-stubs/Chem/Descriptors.pyi

```diff
@@ -19,31 +20,104 @@ _descList: Incomplete
 
 def _setupDescriptors(namespace) -> None: ...
 
-MolWt: Incomplete
+MolWt: Callable
```

Comment:
From the doc comment, it looks like this is meant to be
```suggestion
MolWt: Callable[[Mol], float]
```

Reply:
I completely agree.